game/external/RUNME.py

```diff
@@ -0,0 +1,18 @@
+import os
+import shutil
+import json
+
+manifest = open("manifest.txt", "r")
```

Comment:
any reason this is a `.txt` and not `.json`?

Seems like it holds json values, so I'm not sure why we're not keeping the filetype as that